### ISSUE TITLE
Is the documentation on az keyvault secret show up to date?

### ISSUE THREAD

**Issue by chadcarlton** [Azure/azure-cli#10901]
usage: az keyvault secret show [-h] [--verbose] [--debug]
                               [--output {json,jsonc,table,tsv,yaml,none}]
                               [--query JMESPATH]
                               [--subscription _SUBSCRIPTION] --vault-base-url
                               VAULT_BASE_URL --secret-name SECRET_NAME
                               --secret-version SECRET_VERSION

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 39cfb8ed-c246-2ffb-9dff-070b90c18b21
* Version Independent ID: c09c1597-1b6c-b0a2-b3fc-bdc16764d096
* Content: [az keyvault secret](https://docs.microsoft.com/en-us/cli/azure/keyvault/secret?view=azure-cli-latest#az-keyvault-secret-show)
* Content Source: [src/azure-cli/azure/cli/command_modules/keyvault/_help.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/keyvault/_help.py)
* Service: **key-vault**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by yonzhan** [comment]
@jiasli  add to S163.

**Comment by subhasishdc** [comment]
@chadcarlton : Saw this same issue which you mentioned in the "az keyvault secret show" command as well. However, after doing some digging into the documentation and the az cli help, looks like the parameters are clearly explained. I will provide examples below:

`schowdhu-mn2:~ schowdhu$ az keyvault certificate show
incorrect usage: --id ID | --vault-name VAULT --name NAME [--version VERSION]`

The `az keyvault certificate show --help` commands has this part:

Id Arguments
    --id           : Id of the certificate.  If specified all other 'Id' arguments should be
                     omitted.
    --name -n      : Name of the certificate. Required if --id is not specified.
    --vault-name   : Name of the key vault. Required if --id is not specified.
    --version -v   : The certificate version. If omitted, uses the latest version.

I think it clearly mentions that the required arguments are either `--id` or `--name AND --vault-name` if `--id` is not specified. The documentation also specifies the same: https://docs.microsoft.com/en-us/cli/azure/keyvault/certificate?view=azure-cli-latest#az-keyvault-certificate-show

The behavior and documentation is same for the `az keyvault secret show` as well.

Does this answer your question?

**Comment by bim-msft** [comment]
Hi @subhasishdc thank you for your informative comment!

I think the user's concern is the difference between prompt messages with/without interactive mode:
![image](https://user-images.githubusercontent.com/3250203/84004352-fd215080-a99d-11ea-9591-70f94691a535.png)
I tried to figure out why this happens but nothing found until now. As this issue has low priority and "az interactive" is nearly out of maintenance so I moved this issue to our backlog.

Feel free to contact me if you have any question on this.
